content/fr/elao/forum-php-2015.md

```diff
@@ -0,0 +1,81 @@
+language:           "fr"
+thumbnail:          "/images/posts/thumbnails/kafeine.png"
+header_img:         "/images/posts/headers/elephpant_elao_family.jpg"
+tags:               ["Conférence", "ForumPHP"]
+categories:         ["conference"]
+
+author_username:    "rhanna"
+---
+
+Le Forum Php de l'AFUP (Association Française des Utilisateurs de PHP) a réuni au Beffroi de Montrouge du beau monde
+cette année, des membres de la Core team du développement de Php et le créateur du langage, le groenlandais Rasmus
+Lerdorf.
+Et pour cause ! On fête les 15 ans de l'association et des 20 ans du langage. Et cerise sur le gâteau, on fête
+également la sortie de php7, une version majeure déjà plébiscitée pour les gains en performance par rapport à php5. 
+
+Point d'orgue de la keynote d'ouverture, il a été diffusé en avant première mondiale la video "Haphpy Birthday",
+projet collaboratif animé entre autres par notre ami Julien.
+Un montage de vidéos et de photos provenant du monde entier a célébré l'amour des développeurs pour le langage php.
+Oui en ces temps obscurs, célébrons l'amour !
+L'amour pour le langage mais aussi des valeurs de partage véhiculées par ce langage Open Source.
+Finalement on a la définition ultime de l'acronyme PHP : People Helping People.
+
+## Les confs auxquelles nous avons assisté
+
+Il y avait 3 tracks de conférences étendus sur deux journées et on a dû parfois faire des choix cornelliens entre des
+sujets qui nous intéressent.
+Voici notre retour sur les conférences auxquelles nous avons assisté.
+
+### Taylorisme de la qualité logicielle par Jean-François Lépine
+
+Il s'agit d'un retour d'expérience sur l'industrialisation des développements.
+Nous retenons qu'il est inutile de vouloir tout automatiser.
+Il faut surtout automatiser les process qui sont à risque.
+De plus ne pas laisser la technique résoudre tous les problèmes.
+L'Humain est au cœur de notre métier. Dans une équipe, il faut des meneurs, des gens motivés qui vont porter un projet
+ou des évolutions, que ce soit techniques ou en terme d'organisation. 
+
+### Comment Ansible et Docker changent nore environnement de mise en production par Simon Constans et Maxime Thoonsen (Theodo)
+
+Sur des projets courts de quelques semaines à quelques mois, il est important de ne pas perdre de temps pour initier la
+production. Cette présentation nous a montré comment tirer parti de Ansible et Docker pour être "up" en production en
+très peu de temps.
+
+### Scrum... et après ? par Yvan Wibaux et Olivier Madre (Evaneos)
+
+Un des cofondateurs, un architecte et un Developpeur d'Evaneos nous ont présenté comment ils ont su s'auto-re-organiser
+à un moment critique où de nombreux développeurs sont arrivés en peu de temps et que la startup avait une croissance et
+une demande en nouvelles fonctionnalités assez énormes.
+* Organisation physiques des bureaux
+* Gestion de projet (abandon de Jira, Trello, post-it...)
```

Comment:
Réexprimer cette idée: (abandon de Jira, Trello, post-it...)
Ici on comprend qu'ils ont abandonné à la fois Jira, Trello et les post-it